_posts/_posts/2020-02-06-blog-pdksync_GemTesting.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: post
+title: "Gem Testing with pdksync"
+date: 2020-02-03 14:00:00
+author: sheenaajay
+published: false
+categories:
+  - testing
+  - walkthrough
+tags:
+  - pdksync
+  - modules
+  - gem testing
+---
+> Introduction:
+>
+pdksync is a tool that enables you to run a pdk update against a set of defined modules. The tool checks out a module, runs the most recent version of pdk update then proceeds to create the PR for you with the changes. This is the automated tool for ensuring modules can quickly adopt changes introduced by updates to the PDK and PDK template.
```

Comment:
pdksync enables you to do a lot more than just pdk update against a set of defined modules

Reply:
Sure.Thank you.